cmd/root_args.go

```diff
@@ -51,6 +51,12 @@ var envMapString = map[*string]boundEnvVar[string]{
 		Env:         helpers.Ptr("DYNAMIC_PROMOTION_KEY"),
 		Default:     helpers.Ptr("gitops-promotion-path"),
 	},
+	&feedbackCommitStatusContext: {
+		Name:        "feedback-commit-status-context",
+		Description: "The context key to use when pushing the commit status to the repository. Supported placeholders: {source}, {target}",
+		Env:         helpers.Ptr("FEEDBACK_COMMIT_STATUS_CONTEXT"),
+		Default:     helpers.Ptr("gitops/promotion :: {source} → {target}"),
```

Comment:
Given that we're constrained in the number of characters that GitHub will display before it truncates the string, I suggest we minimise the default (and keep it easy to type without having to revert to UTF-8):
```suggestion
		Default:     helpers.Ptr("{source}/{target}"),
```